go.mod

```diff
@@ -37,3 +37,5 @@ require (
 	k8s.io/client-go v0.19.6
 	k8s.io/utils v0.0.0-20200729134348-d5654de09c73
 )
+
+replace github.com/Microsoft/hcsshim => github.com/projectcalico/hcsshim v0.8.9-calico
```

Comment:
Why we need to make this change?

Reply:
The hcsendpoint `ApplyPolicy` function signature has a new function parameter `requestType RequestType` in v0.8.9: https://github.com/microsoft/hcsshim/blob/v0.8.9/hcn/hcnendpoint.go#L356

Since the node repo also imports our fork of hcsshim v0.8.9 the code using the old signature won't compile.